x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -114,18 +113,18 @@ export const MisconfigurationFindingsDetailsTable = memo(
       }
     };
 
-    const navToFindings = useNavigateFindings();
+    const navUrlParams = useGetNavigationUrlParamsFindings();
 
-    const navToFindingsByRuleAndResourceId = (ruleId: string, resourceId: string) => {
-      navToFindings({ 'rule.id': ruleId, 'resource.id': resourceId });
+    const getFindingsPageUrlFilteredByRuleAndResourceId = (ruleId: string, resourceId: string) => {
+      return navUrlParams({ 'rule.id': ruleId, 'resource.id': resourceId });
     };
 
-    const navToFindingsByName = (name: string, queryField: 'host.name' | 'user.name') => {
+    const getFindingsPageUrl = (name: string, queryField: 'host.name' | 'user.name') => {
```

Comment:
this introduces a bug to metering - the click will be counted on each re-render even if the user doesn't navigate to the link. we still need to meter only on `click` event